app/editor/src/App.tsx

```diff
@@ -23,6 +23,10 @@ function App() {
     });
   }, []);
 
+  React.useEffect(() => {
+    ReactTooltip.rebuild();
```

Comment:
I don't understand how this would work?  It'll get called one time at the beginning of the app.  If the tooltip is set as `undefined` shouldn't we just set it to `''`?

Reply:
I think it might not be the issue I thought it was but rather that in some cases the tooltip is rendered before the element that uses it which results in nothing being bound. If we are going to leave the tooltip at the app.tsx level I believe we will need this. However; I got it to work moving <ReactTooltip /> within the components themselves that use tooltips